test/unit/Vault/Deposit.t.sol

```diff
@@ -553,6 +555,56 @@ contract VaultDepositTest is UnitBaseSetup, BrokenToken {
     vm.stopPrank();
   }
 
+  /* ============ Sweep ============ */
+  function testSweep() external {
+    vm.startPrank(alice);
+
+    uint256 _amount = 1000e18;
+    underlyingAsset.mint(alice, _amount);
+
+    underlyingAsset.transfer(address(vault), _amount);
+
+    vm.stopPrank();
+
+    vm.startPrank(bob);
+
+    vm.expectEmit();
+    emit Sweep(bob, _amount);
+
+    vault.sweep();
+
+    assertEq(vault.balanceOf(alice), 0);
+
+    assertEq(twabController.balanceOf(address(vault), alice), 0);
+    assertEq(twabController.delegateBalanceOf(address(vault), alice), 0);
+
+    assertEq(vault.balanceOf(bob), 0);
+
+    assertEq(twabController.balanceOf(address(vault), bob), 0);
+    assertEq(twabController.delegateBalanceOf(address(vault), bob), 0);
+
+    assertEq(underlyingAsset.balanceOf(address(yieldVault)), _amount);
+    assertEq(yieldVault.balanceOf(address(vault)), _amount);
+    assertEq(yieldVault.totalSupply(), _amount);
+
+    assertEq(vault.totalSupply(), 0);
+    assertEq(vault.availableYieldBalance(), _amount);
+
+    vm.stopPrank();
```

Comment:
After reviewing a number of your BDD .feature tests, I think you're better off refactoring the test itself to be more expressive rather than maintaining a separate document. Let's talk more about this.